src/Symfony/Component/Serializer/Normalizer/CacheableSupportsMethodInterface.php

```diff
@@ -19,6 +19,8 @@
  * supports*() methods will be cached by type and format.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ *
+ * @deprecated since symfony/serializer 6.1, return CacheableSupport from the supports*() method instead
```

Comment:
As a new deprecation is added, you must add an entry in the `UPGRADE-6.2.md` file and add another entry mentioning that deprecation in the `CHANGELOG.md` file